src/modules/submodules/consensus.py

```diff
@@ -352,3 +355,7 @@ def is_main_data_submitted(self, blockstamp: BlockStamp) -> bool:
     @abstractmethod
     def is_contract_reportable(self, blockstamp: BlockStamp) -> bool:
         """Returns true if contract is ready for report"""
+
+    @abstractmethod
+    def check_sanity(self, blockstamp: ReferenceBlockStamp) -> bool:
+        """Check if collected report data is valid."""
```

Comment:
The data is valid, but the build_output is unexpected and need to be checked manually

Reply:
rephrased